packages/kbn-generate-csv/index.ts

```diff
@@ -8,4 +8,3 @@
 
 export { CsvGenerator } from './src/generate_csv';
 export { CsvESQLGenerator, type JobParamsCsvESQL } from './src/generate_csv_esql';
-export type { CsvPagingStrategy } from './types';
```

Comment:
This interface shouldn't really be imported from this package, because it raises the potential for many circular dependency issues